features/support/content_register.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../../test/support/content_register_helpers'
+
+Before do
+  stub_content_register
```

Comment:
This is really stubbing content register with policies, let's call it that so there's no confusion as to the scope of the stubbing.